app/src/main/java/ca/klapstein/baudit/activities/PatientHomeActivity.java

```diff
@@ -30,6 +33,9 @@
  */
 public class PatientHomeActivity extends AppCompatActivity implements HomeView {
 
+    public static final String PROBLEM_POSITION_EXTRA = "problemPosition";
```

Comment:
I would move these fields into the `ProblemActivity` as that's whats looking for and using these fields. Its better that it controls/contains it.

Reply:
Agreed